templates/subject/search.html.twig

```diff
@@ -13,13 +12,77 @@
  {% endblock %}
 
 {% block body %}
+
+    <form method="get" action="{{ path('subject_search') }}" class="form-horizontal">
```

Comment:
Any reason not to use `partial/search-form.html.twig`?

Reply:
I copied it from material/search.html.twig.